src/test/java/com/learning/selenium/app/Application.java

```diff
@@ -0,0 +1,47 @@
+package com.learning.selenium.app;
+
+import com.learning.selenium.pages.*;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Application {
+
+    public WebDriver driver;
+    private PasteMainPage pasteMainPage;
+    private GoogleCloudMainPage googleCloudMainPage;
+    private GoogleCloudHeader googleCloudHeader;
+    private GoogleCloudSearchResultsPage googleCloudSearchResultsPage;
+    private GoogleCloudPricingCalcPage googleCloudPricingCalcPage;
+
+    public Application() {
+        driver = new ChromeDriver();
```

Comment:
https://github.com/ivanrightman/SeleniumWebDriverPracticalTasks/pull/1/files#r958182104

Reply:
https://github.com/ivanrightman/SeleniumWebDriverPracticalTasks/pull/1/files#r959357208

Reply:
WebDriver needs to be initialised in the base test, because "application" layer logic should know nothing about the process of driver creation/initialisation/tear-down

Reply:
Updated in Task2